handbook/engineering/README.md

```diff
@@ -669,25 +672,28 @@ After it's in a release formally, the bug should be treated like any other piece
 #### Fast track for Fleeties
 Fleeties do not have to wait for QA to reproduce the bug. If you're confident it's reproducible, it's a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
 
-#### Release testing
-When a release is in testing, QA should use the the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
+### Release testing
+When a release is in testing, QA should use the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
 
-In the #help-qa channel, product may decide whether the bug is a release blocker. When a release-blocking bug is found, product is responsible for communicating the delay to company stakeholders. 
+In the #help-qa channel, product may decide whether the bug is a critical or release-blocking. When a critical bug is found, product is responsible for following the [critical bug notification process](https://fleetdm.com/docs/contributing/releasing-fleet#critical-bug-notification-process) below. When a release-blocking bug is found, product is responsible for communicating any delays to company stakeholders. 
 
-Release blockers include:
-1. Critical bugs (defined below)
-2. New functionality that impacts previous stable functionality
-3. Incomplete features as defined or implied in the specs
+Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard [bug process](https://fleetdm.com/docs/contributing/releasing-fleet#bug-process). 
 
-Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard bug process (defined above).
+### Release-blocking bugs
+A release-blocking bug is identified by the `~release blocker` label. It becomes a release-blocking bug if:
+* New functionality that impacts previous stable functionality
+* Incomplete features as defined or implied in the specs
+* It does not meet the criteria of a critical bug
```

Comment:
What is the implication of it being release blocking only if it's not a critical bug? If it is a critical bug, then does it still block the release?

Reply:
I think that bug would/should have met the definition of critical. If it's a user workflow we support (end user can see their own vulnerable software), and it can no longer be done anymore, no workarounds, we need to warn the community. It demonstrates we're on top of things. 

Not sure we need to tie a bug to a release that doesn't touch the affected area. Either the bug is important enough to do now (critical), or it is not important and we can prioritize fixing it in the next release as per the standard process.

Reply:
We've started using the `~release blocker` label on some bugs to indicate they are particularly important, but to your point, that's not the most straightforward approach. If we remove the concept of a `~release blocker`, we'll have fewer new bugs coming into the sprints, which will help us close the older bugs already in the sprint. 

We will still have [bugs like this](https://github.com/fleetdm/fleet/issues/12113) that we need to fix quickly to support customer needs; the priority of those can be communicated on a case-by-case basis and without the use of a special label.

Will revise.

Reply:
Removed.